qcdb/qcvars/glossary.py

```diff
@@ -756,6 +756,19 @@ def define_ex_transition_qcvars(max_root, mtd, sym):
                 The total energy of EOM-{} in {} symmetry from 0 to root {}""".format(mtd, sym, i)
         }
 
+def define_tddft_roots_qcvars(max_root, sym):
+    sym = []
+
+    for i in range(max_root):
+        qcvardefs['TDDFT ROOT {} EXCITATION ENERGY - {} SYMMETRY'.format(i+1, sym)] = {
+                'units' : 'Eh',
+                'glossary' : """ The excitation energy of time-dependent DFT in {} symmetry from 0 to root {}""".format(sym, i+1)
+                }
+
+        qcvardefs['TDDFT ROOT {} EXCITED STATE ENERGY - {} SYMMETRY'.format(i+1, sym)] = {
```

Comment:
this should be `EXCITED STATE` --> `TOTAL` and the glossary entry edited, right? Something like "The total energy of the {}-th excited state (GS is root 0) by time-dependent DFT of symmetry {}"